figures/malformed-websocket-test-sequence.mmd

```diff
@@ -0,0 +1,8 @@
+sequenceDiagram
+    TestRunner ->> Client : Create
+    activate Balancer
+    Client ->> Balancer : Connect
```

Comment:
The activation extends way too far up. It should should start at this `Connect` line. Use the `+` and `-` syntax.

```
sequenceDiagram
    Alice->>+John: Hello John, how are you?
    John-->>-Alice: Great!
```

renders as 

```mermaid
sequenceDiagram
    Alice->>+John: Hello John, how are you?
    John-->>-Alice: Great!
```